deno/v8/payloads/interactions.ts

```diff
@@ -115,17 +115,24 @@ export interface APIApplicationCommandInteractionData {
 /**
  * https://discord.com/developers/docs/interactions/slash-commands#interaction-applicationcommandinteractiondataoption
  */
-export interface APIApplicationCommandInteractionDataOption {
-	name: string;
-	/**
-	 * The value returned here depends on the `ApplicationCommandOptionType` type of the option with the name
-	 * that matches this interface's `name`.
-	 *
-	 * You will need to manually cast this to the appropriate type based on the returned data
-	 */
-	value?: unknown;
-	options?: APIApplicationCommandInteractionDataOption[];
-}
+export type APIApplicationCommandInteractionDataOption =
+	| { name: string; options: APIApplicationCommandInteractionDataOption[] }
```

Comment:
🤔 is this assuming that if there's no `type` then it's always a subcommand/group? currently, all options for commands sent on mobile don't have a `type` but in the future, all options will have a `type` afaik

Reply:
That first line shouldn't be there. For the mobile thing, are you suggesting making `type` nullable for now?

Reply:
or ig just remove that line and wait until it's fully released to merge this?

Reply:
> That first line shouldn't be there.
Fixed this in 69ab92a

For the mobile `type` thing, someone said it would be rolled out this week, so it's probably not worth removing for just a few days.